<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar Consultas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    background-color: rgb(0, 0, 0);
    font-family: Arial;
    color: rgb(255, 255, 255);
}

QMenuBar{
    text-decoration: underline rgb(255, 255, 255);
}

QLabel{
    font: 75 18pt;
}

QLineEdit,QComboBox,QDoubleSpinBox,QSpinBox{
    color: rgb(0, 0, 0);
    background-color: rgb(255, 255, 255);
    font: 75 14pt &quot;Arial&quot;;
}

QPushButton{
  color: #fff !important;
  font-size: 20px;
  font-weight: 500;
  background: #318aac;
  border: 2px solid;
  border-color: #318aac;
  position: relative;
}

QPushButton:before {
  content:&quot;&quot;;
  position: absolute;
  top: 0px;
  left: 0px;
  width: 100%;
  height: 0px;
  background: rgba(255, 255, 255, 0.1);
  transition: all 1s ease;
}

QPushButton:hover:before {
  height: 100%;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;Editar Consultas&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Editar Consultas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Id de Consulta que quiere Editar:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_ConsultaId">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Id del Medico:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_MedicoId">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Id del Paciente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_PacienteId">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Motivo de Consulta:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_MotivoConsulta">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Visita:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_Pruebas">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_Diagnostico">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>150</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="input_Tratamiento">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>150</y>
      <width>131</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>120</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Pruebas Realizadas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Diagnóstico</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>Tratamiento</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_Buscar">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabla_Consultas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1021</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Id Consulta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre Médico</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Médico</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombre Paciente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Paciente</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha De Consulta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Motivo de Consulta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pruebas Realizadas</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Diagnóstico</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tratamiento</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="input_Aceptar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>31</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dia&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>250</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>75</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2020</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2026</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2027</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2028</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2029</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2030</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2031</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2032</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2033</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2034</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2035</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2036</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2037</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2038</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2039</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2040</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Año&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_fecha">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1042</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionRegresar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRegresar">
   <property name="text">
    <string>Regresar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
